eggbox_potential_sampler/random_sampling_mco/mco.py

```diff
@@ -50,79 +48,12 @@ def run(self, model):
             trial_position = np.random.rand(len(model.parameters))
 
             kpis = single_point_evaluator.evaluate(trial_position)
+            weights = []
```

Comment:
I'd use the same mechanism from below rather than a for loop:
```
weights = [1 / len(kpis)] * len(kpis)
```

Reply:
Done, as well as editing unit test that was failing as `len(kpis) == 0`. This behaviour is not expected to happen outside the test environment, as it would indicate that the `BaseMCOModel` does not have any KPIs assigned.